admin/app/helpers/solidus_admin/components_helper.rb

```diff
@@ -5,5 +5,9 @@ module ComponentsHelper
     def component(name)
       SolidusAdmin::Config.components[name]
     end
+
+    def search_filter_params
+      request.params.slice(:q, :page)
```

Comment:
It seems like the tests are passing so I assume this is working as expected. What's the general consensus on the impact of the redirect vs the turbo refresh on the modal?

Reply:
I think it's acceptable to redirect as soon as the parameters are preserved. At the end of the day, this is a great trade-off between UX and semantic.

Reply:
>  What's the general consensus on the impact of the redirect vs the turbo refresh on the modal?

> I think it's acceptable to redirect as soon as the parameters are preserved. At the end of the day, this is a great trade-off between UX and semantic.

It does not have any impact, because with Turbo the result is exactly the same. Turbo refresh reloads the whole page, while a redirect from a turbo frame just replaces the body (preserving the head and with that all global script and style tags). There is no notable performance impact. Quite the opposite. Also, it is much more "correct" to redirect after a form submit. The browser needs to fetch the result of the POST, which is indicated with the 303 (see other) response code.